lib/admin/WP_Auth0_Admin_Appearance.php

```diff
@@ -290,10 +290,11 @@ public function render_extra_conf( $args = [] ) {
 	 * @return array
```

Comment:
Everything in this file is moved from somewhere else. No functional changes, validation will be addressed in a separate PR.